### PR TITLE
Handle no response

### PR DESCRIPTION
**Pull request by peteruithoven**
For example timeouts and network errors. In these cases the generated Error instance doesn't contain a response object.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ class Superlogin extends EventEmitter2 {
 
 			// If there is an unauthorized error from one of our endpoints and we are logged in...
 			if (checkEndpoint(error.config.url, config.endpoints) &&
-				error.response.status === 401 && this.authenticated()) {
+				error.response &&	error.response.status === 401 && this.authenticated()) {
```
Comment (micky2be) [inline src/index.js]: white space problem

File: src/index.js [hunk 2]
```diff
@@ -362,7 +362,7 @@ class Superlogin extends EventEmitter2 {
 			})
 			.catch(err => {
 				this._onLogout(msg || 'Logged out');
-				if (err.response.data.status !== 401) {
+				if (!err.response || (err.response && err.response.data.status !== 401)) {
```
Comment (micky2be) [inline src/index.js]: No need for the double check. The following should be enough
`(!err.response ||  err.response.data.status !== 401)`

**Review by micky2be — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (peteruithoven) [reply src/index.js]: Good point!

**Comment by peteruithoven** [comment]
Thanks guys! 
I'm hoping these (#53, the safari private mode fix, #54, #55, #56) where all the fixes I wanted to get done for the time being, so could the last changes be included into a new release?

**Comment by micky2be** [comment]
No problem, gonna make a new release asap

**Comment by peteruithoven** [comment]
@micky2be friendly reminder for that release.

**Comment by micky2be** [comment]
Done, thanks for the reminder

**Comment by peteruithoven** [comment]
Thanks! :)

**Pull request merged**